### PR TITLE
costa: make boa and datacook be global

### PR DESCRIPTION
**Pull request by FeelyChau**
The script is not adaptive for now.

### CODE REVIEW

File: packages/core/src/runtime.ts [hunk 1]
```diff
@@ -42,14 +42,6 @@ export type DataCookModule = typeof Datacook;
...
-   * The boa module, the bridge between node and python.
-   */
-  boa: FrameworkModule;
-  /**
-   * DataCook module
-   */
-  dataCook: DataCookModule;
   /**
    * This function can import a node module from framework. if the module not exists,
    * an error will be thrown.
```
Comment (yorkie) [inline packages/core/src/runtime.ts:47]: importJS and importPY could be removed :)

**Comment by yorkie** [comment]
Boa is improper as a global object because we don't expect it as a Web API.

File: packages/costa/src/index.ts [hunk 2]
```diff
@@ -11,14 +11,25 @@ import {
...
 import Debug from 'debug';
 import { importFrom } from './utils';
 
 const debug = Debug('costa.runnable');
 
+declare global {
+  // eslint-disable-next-line no-var
+  var boa: typeof BOA;
+  // eslint-disable-next-line no-var
+  var datacook: typeof Datacook;
```
Comment (yorkie) [inline packages/costa/src/index.ts:26]: Shall we make datacook to extend the global, thus user could use it just like:

```js
return Dataset.makeDataset({ trainData, testData }, meta);
```

**Comment by yorkie** [comment]
That's great if developers still could import Python package via ESM:

```js
import * as np from 'py:numpy';
```

See https://github.com/imgcook/boa/blob/main/tests/es-module-loaders/py/test-esm-loader-custom.mjs.

**Comment by yorkie** [comment]
Any updates @FeelyChau?

**Comment by FeelyChau** [comment]
This PR could be closed, `boa` and `DataCook` could be required at runtime by excluding then when bundle.

**Pull request closed without merging**